sycl/test/sub_group/load_store.cpp

```diff
@@ -22,7 +22,7 @@ template <typename T, int N> class sycl_subgr;
 using namespace cl::sycl;
 
 template <typename T, int N> void check(queue &Queue) {
-  const int G = 1024, L = 64;
+  const int G = 1024, L = 128;
```

Comment:
Why did you increase the workload here?

Reply:
The default subgroup size on avx512 is 16, so the minimal local range is 16 * 8, otherwise load/store to  LocalMem will be out of range